tests/api/test_apis.py

```diff
@@ -46,3 +47,9 @@ def test_item_exists(testdata):
 def test_circulation_item_not_exist(testdata):
     """Test return False if item does not exist."""
     assert not item_exists("not-existing-item-pid")
+
+
+def test_patron_exists(testdata):
+    """Test return True if item exists."""
+    first_patron_pid = testdata["loans"][0][Patron.pid_field]
```

Comment:
Patron is not a real record, so I think `Patron.pid_field` attribute does not exist (unless you add it).

I would also add a test for the negative, when it does not exist.

Reply:
Yes you are right :)

Reply:
All done thanks